Cloudinary/Features/Helpers/CLDTranformation.swift

```diff
@@ -622,6 +622,20 @@ import Foundation
     open func setRadius(_ radius: String) -> Self {
         return setParam(TransformationParam.RADIUS, value: radius)
     }
+    /**
+     Support an array value for radius
+      
+      - parameter radius:    - The radius to apply- an array of values
+      
+      - returns:             The same instance of CLDTransformation.
+      */
+    
+    @objc(setRadiusFromArray:)
+     @discardableResult
+     open func setRadius(_ radius: [Any]) -> Self {
+       return setRadius(radius.map{String(describing: $0) }.joined(separator:":"))
```

Comment:
Formatting is a bit off (spaces after braces, param names, etc). Apart from that everything is 100%

Reply:
Thanks